static/bidder-info/nativo.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://exchange.postrelease.com/esi?ntv_epid=7"
+maintainer:
+  email: "prebiddev@nativo.com"
```

Comment:
I've sent an email to this address to confirm it is correct. Please respond to the email with a "received" message.

Reply:
It is correct. I've just replied to the email.

Reply:
mail confirmed